<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionWindow</class>
 <widget class="QDialog" name="TransactionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przelewy</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>401</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="transactionLabel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight SemiConde</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Przelew</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="whichAccountLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Konto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="reciverLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Odbiorca</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="reciverInput">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="reciverAccNumbLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Konto odbiorcy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="reciverAccNumberInput">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>26</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tytuł</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleInput">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>26</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="amountLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kwota</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amountInput">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>26</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Wyślij</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gobackButton">
        <property name="text">
         <string>Wróć</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
